CHANGELOG.md

```diff
@@ -1,66 +1,54 @@
+- Revert "Add direct support for bun, pnpm and yarn" -- issues with pnpm
 
-  * Revert "Add direct support for bun, pnpm and yarn" -- issues with pnpm
+# v2.1.0 / 2024-04-04
 
-v2.1.0 / 2024-04-04
-==================
+- Add support for different package managers (thank you @andershagbard)
 
-  * Add support for different package managers (thank you @andershagbard)
+# v2.0.1 / 2024-02-14
 
-v2.0.1 / 2024-02-14
-==================
+- Don't pass the `--dev-preview` flag for Shopify CLI versions >= 3.55.0
 
-  * Don't pass the `--dev-preview` flag for Shopify CLI versions >= 3.55.0
+# v2.0.0 / 2024-01-05
 
-v2.0.0 / 2024-01-05
-===================
+- Theme Check 2.0 and Shopify CLI 3.50+ support
+- Remove support for Ruby Theme Check (v1)
 
-  * Theme Check 2.0 and Shopify CLI 3.50+ support
-  * Remove support for Ruby Theme Check (v1)
+# v1.3.1 / 2024-01-05
 
-v1.3.1 / 2024-01-05
-===================
+- Merge pull request #29 from snacsnoc/update-checkout-v3-node16-compatibility
+- Update ci.yml to use checkout@v3
+- Update README.md to reflect upgrade to checkout@v3
 
-  * Merge pull request #29 from snacsnoc/update-checkout-v3-node16-compatibility
-  * Update ci.yml to use checkout@v3
-  * Update README.md to reflect upgrade to checkout@v3
+# v1.3.0 / 2023-11-28
 
-v1.3.0 / 2023-11-28
-===================
+- Run Theme Check with the most updated version of objects, filters, and tags ([#19](https://github.com/Shopify/theme-check-action/pull/19))
 
-  * Run Theme Check with the most updated version of objects, filters, and tags ([#19](https://github.com/Shopify/theme-check-action/pull/19))
-
-v1.2.0 / 2022-02-28
-===================
+# v1.2.0 / 2022-02-28
 
 ## Features
 
-  * Add support to only annotate files with diffs in them ([#3](https://github.com/shopify/theme-check-action/issues/3), [#9](https://github.com/shopify/theme-check-action/issues/9))
-  * Add the `base` input argument
+- Add support to only annotate files with diffs in them ([#3](https://github.com/shopify/theme-check-action/issues/3), [#9](https://github.com/shopify/theme-check-action/issues/9))
+- Add the `base` input argument
 
-v1.1.1 / 2022-02-28
-===================
+# v1.1.1 / 2022-02-28
 
-  * Throttle the requests to respect the rate limits ([#8](https://github.com/shopify/theme-check-action/issues/8))
-  * GH Actions Checks: Submit the checks in batches ([#7](https://github.com/shopify/theme-check-action/issues/7))
+- Throttle the requests to respect the rate limits ([#8](https://github.com/shopify/theme-check-action/issues/8))
+- GH Actions Checks: Submit the checks in batches ([#7](https://github.com/shopify/theme-check-action/issues/7))
 
-v1.1.0 / 2022-02-25
-===================
+# v1.1.0 / 2022-02-25
 
 ## Features
 
-  * Add GitHub Annotation support
-  * Add the `token` input argument
+- Add GitHub Annotation support
+- Add the `token` input argument
 
-v1.0.0 / 2021-07-08
-===================
+# v1.0.0 / 2021-07-08
 
-  * Initial release
+- Initial release
```

Comment:
Can we revert the stylistic changes, IIRC I used `git-extras`'s `git-changelog` for this.

Reply:
Maybe don't run prettier on the md files? Prettier on CODE_OF_CONDUCT is a shouldn't be in this PR

Reply:
For me, it makes sense to have it run on .md files, as it provides a common way to write it.

I can see it makes sense to omit the CODE_OF_CONDUCT file, as I assume that is a default file from Shopify.